### PR TITLE
App amends

### PR DESCRIPTION
**Pull request by matt-harris**
- Added Netlify.
- Refactored App to use Styled Components.
- Refactored state to use declarative style. General tidy up.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -53,33 +68,36 @@ const App = () => {
...
-    fetchData(country);
+  function handleCountryChange(country) {
     setCountry(country);
   }
 
   useEffect(() => { fetchData() }, []);
   useEffect(() => { fetchDailyData() }, []);
-  useEffect(() => { fetchCountryData() }, []);
+  useEffect(() => { fetchCountryNames() }, []);
+  useEffect(() => { fetchData(country) }, [country]);
```
Comment (geeogi) [inline src/App.js:89]: nice, this should run on first render with an `undefined` country too, so perhaps line 86 isn't needed?

**Comment by geeogi** [comment]
React select seems to work really well!

File: src/App.js (hunk 1 above)
Comment (matt-harris) [reply src/App.js:89]: Yes its not needed 👍

**Comment by matt-harris** [comment]
Thanks George!

**Pull request merged**